src/web/components/ApiKeyManagement/KeyEditDialog.scss

```diff
@@ -1,5 +1,8 @@
 .key-edit-dialog {
   .unapproved-roles-message {
-    color: var(--theme-error);
+    margin-top: 10px;
+    ul {
+      margin: 0px;
```

Comment:
I feel like this could possibly do with some vertical spacing at the bottom. 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/011c2a99-21fc-4d1a-8758-c42fb1c69fd0)

Maybe this could be achieved by the following? (not tested)
```
margin-top: 0px;
```

Reply:
I just realized I'm not sure what you meant sorry, I assumed you meant shortening the gap between the different parts.

. I think I prefer it with the gap but would be open to removing it.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/ffe06017-3b5c-4870-8cfd-25ebbc142e90)

Though lmk if you meant something else.